.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
I would add this file to `.gitignore` and remove the file from remote since it is an IDE file 😄

Reply:
Exactly, add that file to `.gitignore`, but remember that you need to remove that file from previous commits.